### DISCUSSION TITLE
QR Code Different (Snipe-IT vs other tools)

### DISCUSSION THREAD

**Discussion by myokeeh** [snipe/snipe-it#14854] [Q&A]
If I encode the URL for the asset using a different tool, it is different from one I get generated with Snipe-IT. Why is that?

I tried several tools and they all generate the same QR Code. Is there another piece of information that is encoded with the Snipe-IT version?

**Comment by mauro-miatello** [comment]
I think, but I'm not sure, it depends from the qrcode type
you can set it in admin/barcodes

**Reply by myokeeh** [reply]
It is set to QRCODE.
